UnityProject/Assets/Scripts/Health/Living/LivingHealthBehaviour.cs

```diff
@@ -143,6 +146,7 @@ public override void OnStartServer()
 		DNABloodTypeJSON = JsonUtility.ToJson(DNABloodType);
 		bloodSystem.SetBloodType(DNABloodType);
 		base.OnStartServer();
+		registerPlayer = GetComponent<RegisterPlayer>();
```

Comment:
you know, not all LivingHealthBehaviours are players. you'll get NREs with animals if you try to calculate stun below

Reply:
Moved all that logic to RegisterPlayer, because of it's dependency on logic there like LayDown(), GetUp() and UpdateMe().